object-cache.php

```diff
@@ -527,6 +533,58 @@ function get_multi( $groups ) {
 		return $return;
 	}
 
+	public function get_multiple( $ids, $group = 'default', $force = false ) {
+		$mc = $this->get_mc( $group );
+
+		$no_mc = in_array( $group, $this->no_mc_groups );
+
+		$uncached_keys = array();
+
+		$return = array();
+		$return_cache = array();
+
+		foreach ( $ids as $id ) {
+			$key = $this->key( $id, $group );
+
+			if ( isset( $this->cache[ $key ] ) && ( ! $force || $no_mc ) ) {
+				$value         = $this->cache[ $key ]['found'] ? $this->cache[ $key ]['value'] : false;
```

Comment:
There is a nice feature: when `set()` fails, we may end up with some value and `found` [set to `false`](https://github.com/Automattic/wp-memcached/blob/ec129864ad26dc72519b8e32a01f3be391d21251/object-cache.php#L682).

I chose to return `false` because the value is not in Memcached. Returning something else may break add/increment scenarios.